File: src/2027.minimum-moves-to-convert-string.2154/solution.ts
function minimumMoves (s: string): number {
    let result = 0;
    let index = 0;
    while (index < s.length) {
        if (s[index] === 'O') {
            index++;
        } else {
            result++;
            index += 3;
        }
    }

    return result;
}

File: src/2027.minimum-moves-to-convert-string.2154/solution.java
class Solution {
    public int minimumMoves(String s) {
        char[] chars = s.toCharArray();
        int result = 0;
        int index = 0;
        while (index<chars.length){
            if(chars[index] == 'X'){
                result++;
                index += 3;
            }else{
                index++;
            }
            
        }
        return result;
    }
}